dev/package-examples/endpoint-0.0.1/manifest.yml

```diff
@@ -0,0 +1,17 @@
+format_version: 1.0.0
+name: endpoint
+title: Elastic Endpoint
```

Comment:
Are we reusing this package for Phase 1? If so, then this is probably the right name. But if we expect these will break in Phase 1, then it might be important for us to consider whether we adjust the name so that it's clear it's for Elastic Endpoint Security.

Reply:
I expect this to be the same package which is shipped also for v1, so we only have 1 endpoint package.

Reply:
@tonymeehan I believe this package is just an example to get discussion going. I don't think we'll have a package for the phase 0 work. I'm working on the phase 1 package.

Reply:
@tonymeehan yeah this was to start conversations